app/controllers/api/v1/articles_controller.rb

```diff
@@ -0,0 +1,13 @@
+class Api::V1::ArticlesController < ApplicationController
+  rescue_from ActiveRecord::RecordNotFound, with: :not_found
+
+  def show
+    article = Article.find(params[:id])
+    render json: article 
+  end
+
+  def not_found
```

Comment:
move this to the application controller and make sure it is a private method

Reply:
ok